src/gui/widgets/CPULoadWidget.cpp

```diff
@@ -91,10 +91,21 @@ void CPULoadWidget::paintEvent( QPaintEvent *  )
 
 void CPULoadWidget::updateCpuLoad()
 {
-	// smooth load-values a bit
-	int new_load = ( m_currentLoad + Engine::audioEngine()->cpuLoad() ) / 2;
-	if( new_load != m_currentLoad )
+	// Additional display smoothing for the main load-value. Stronger averaging
+	// cannot be used directly in the profiler: cpuLoad() must react fast enough
+	// to be useful as overload indicator in AudioEngine::criticalXRuns().
+	const int new_load = (m_currentLoad + Engine::audioEngine()->cpuLoad()) / 2;
+
+	if (new_load != m_currentLoad)
 	{
+		auto engine = Engine::audioEngine();
+		setToolTip(
+			tr("DSP total: %1").arg(new_load) + " %\n"
+			+ tr(" - Notes and setup: %1").arg(engine->detailLoad(AudioEngineProfiler::DetailType::NoteSetup)) + " %\n"
+			+ tr(" - Instruments: %1").arg(engine->detailLoad(AudioEngineProfiler::DetailType::Instruments)) + " %\n"
+			+ tr(" - Effects: %1").arg(engine->detailLoad(AudioEngineProfiler::DetailType::Effects)) + " %\n"
+			+ tr(" - Mixing: %1").arg(engine->detailLoad(AudioEngineProfiler::DetailType::Mixing)) + " %"
```

Comment:
I think I prefer the percent signs without a space before them, and it's what we currently do elsewhere in the program. Also, it's better to keep text in translatable strings where possible.
```suggestion
			tr("DSP total: %1%").arg(new_load) + "\n"
			+ tr(" - Notes and setup: %1%").arg(engine->detailLoad(AudioEngineProfiler::DetailType::NoteSetup)) + "\n"
			+ tr(" - Instruments: %1%").arg(engine->detailLoad(AudioEngineProfiler::DetailType::Instruments)) + "\n"
			+ tr(" - Effects: %1%").arg(engine->detailLoad(AudioEngineProfiler::DetailType::Effects)) + "\n"
			+ tr(" - Mixing: %1%").arg(engine->detailLoad(AudioEngineProfiler::DetailType::Mixing))
```

Reply:
Oh, that's interesting. In Czech, "10 %" and "10%" are pronounced differently and mean different things. The first one just states "ten percent", as a standalone number, while the second form is used as an adjective. I thought it works the same way in English so I used the same spacing.

So good idea with moving the symbol into the translatable string, since Czech translation should definitely keep the space. :)

Reply:
I never knew that about Czech - it's always interesting to find these things out. Looking at https://en.wikipedia.org/wiki/Percent_sign#Form_and_spacing, it appears the styling varies greatly between languages - some even put the percent sign before, not after, the number. As you say, all the more reason to make it translatable.